### PR TITLE
ISLANDORA-2409: Selectable colorspace for PREVIEW/TN

### PR DESCRIPTION
**Pull request by BrandonMorr**
**JIRA Ticket**: https://jira.duraspace.org/browse/ISLANDORA-2409

# What does this Pull Request do?

Adds a dropdown to the solution pack config UI which allows users to select from a list of colorspaces to use when creating TN/PREVIEW derivatives. The intention of this pull is to allow users to choose a more reliable/consistent colorspace profile for PDF derivatives. The currently hardcoded RGB colorspace appears to create substantially darker derivatives compared to their source file.

# What's new?

* Adds dropdown select to configure colorspace configuration for PREVIEW & TN derivative datastreams
* Profiles include RBG, sRGB, CYMK & Grayscale
* Update hook preserves existing colorspace for older systems

# How should this be tested?

Prior to pulling down the changes, ingest a test PDF (preferably a PDF with light colors on the 1st page) and observe that the PREVIEW and TN datastreams colors are far darker than the source PDF's 1st page's colors.

Pull down the changes and go to the PDF scholar solution pack config UI. Confirm the sRGB colorspace is set for both PREVIEW and TN and submit the form. Ingest another test PDF and observe the PREVIEW and TN datastreams and confirm that they are closer to the original source PDF.

# Additional Notes:

Similar pull request for PDF sol pack: https://github.com/Islandora/islandora_solution_pack_pdf/pull/138

### CODE REVIEW

**Review by qadan — APPROVED** [review]
Acceptably improves from the usage of Adobe RGB colorspace 👍

**Comment by jordandukart** [comment]
Bump @Islandora/7-x-1-x-committers and @dannylamb

File: includes/derivatives.inc
Comment (jonathangreen) [inline includes/derivatives.inc]: Should we be changing the function signature here in case someone is calling this in their own module? 

```php
function islandora_scholar_add_jpg_derivative(AbstractObject $object, $file_uri, $dsid, $width, $height, $force, $source_dsid, $colorspace = NULL)
```

Might be more appropriate?

I don't see any uses in core code, or my own code, so I'm not super concerned, but I thought it was worth mentioning.

**Review by jonathangreen — COMMENTED** [review]
This looks fine to me, just one comment about a changed function signature.

File: includes/derivatives.inc
Comment (BrandonMorr) [reply includes/derivatives.inc]: Definitely agree! Also caught some missing stuff. Thanks for the feedback 👍

Comment (bryjbrown) [reply includes/derivatives.inc]: +1 to @jonathangreen's suggestion.

**Comment by jonathangreen** [comment]
LGTM pending travis. 👍🏻

**Review by jonathangreen — APPROVED** [review]
Start merge ⏱

**Comment by BrandonMorr** [comment]
@jonathangreen merge time? :)

**Pull request merged**
